dask_expr/_collection.py

```diff
@@ -2830,6 +2830,9 @@ def merge(
               on is specified) with other frame's index, preserving the order
               of the calling's one
             - leftsemi: Choose all rows in left where the join keys can be found
+              in right. Won't duplicate rows if the keys are duplicated in right.
+              Drops all columns from right. Only supported with 'cudf' backend.
```

Comment:
I believe `"Only supported with 'cudf' backend."` only applies to "leftanti" in dask-expr.

Reply:
Good catch!